pyartcd/pyartcd/pipelines/ocp4_scan.py

```diff
@@ -65,28 +67,31 @@ async def run(self):
                 comment_on_pr=True,
             )
 
-        elif self.rhcos_inconsistent:
-            self.logger.info('Detected inconsistent RHCOS RPMs:\n%s', self.inconsistent_rhcos_rpms)
+        if self.rhcos_inconsistent or self.rhcos_outdated:
+            if self.rhcos_inconsistent:
+                self.logger.info('Detected inconsistent RHCOS RPMs:\n%s', self.inconsistent_rhcos_rpms)
+            if self.rhcos_outdated:
+                self.logger.info('Detected outdated RHCOS RPMs:\n%s', self.changes.get('rhcos', None))
 
             if self.runtime.dry_run:
                 self.logger.info('Would have triggered a %s RHCOS build', self.version)
                 return
 
             # Inconsistency probably means partial failure and we would like to retry.
             # but don't kick off more if already in progress.
-            self.logger.info('Triggering a %s RHCOS build for consistency', self.version)
-            jenkins.start_rhcos(build_version=self.version, new_build=True, job_name="build")
-            if int(self.version.split(".")[1]) == 19:
-                jenkins.start_rhcos(build_version=self.version, new_build=True, job_name="build-node-image")
+            self.logger.info('Triggering a %s RHCOS build', self.version)
+            group_config = await util.load_group_config(group=f"openshift-{self.version}", assembly="stream")
+            job_name = "build-node-image" if ("layered_rhcos" in group_config['rhcos'] and not self.rhcos_rhel_outdated) else "build"
```

Comment:
for layered_rhcos if update come from rhel repo, then only need to trigger build job